cldoc/nodes/field.py

```diff
@@ -23,6 +23,8 @@ def __init__(self, cursor, comment):
         self.type = Type(cursor.type, cursor=cursor)
 
     def compare_same(self, other):
+        def cmp(a, b):
```

Comment:
I was going to submit a PR to do exactly this. Just wanted to note that the author has actually already defined this new `cmp` function. So instead of defining it here, just add an import at the top of the file,

```python
from ..cmp import cmp
```